opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -49,6 +50,9 @@ impl opentelemetry::logs::LoggerProvider for LoggerProvider {
         attributes: Option<Vec<opentelemetry::KeyValue>>,
     ) -> Logger {
         let name = name.into();
+        otel_info!(target: "opentelemetry-sdk", name: "logger_versioned_creation", signal: "log", 
```

Comment:
target
I think the usage of component name for target looks good. Though we need to discuss further about making it even more specific, like opentelemetry-sdk.log.logemitter?

name
Looks good. We need to develop conventions. (like using "." vs "_" separator).

signal
I don't think this is required. Target+Name should be sufficient.

message
I think we should only promote key/values pairs, and leave the message out, unless there is a strong need.
Lets also use a named field as that is proved much faster : https://github.com/open-telemetry/opentelemetry-rust/pull/2001

Reply:
So, 
target := <component/crate>.<signal>.<module/struct>
name := <operation-name> eg logger_creation
And any unstructured data (if required) as "message" attribute.

So for batch::emit method:

target := opentelemetry_sdk.logs.log_processor
name := batch_emit
event_count := 10 (say)

Reply:
that looks very reasonable for first release.!